stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/utils/extensions/Channel.kt

```diff
@@ -1,11 +1,11 @@
 package com.getstream.sdk.chat.utils.extensions
 
-import io.getstream.chat.android.client.extensions.getUsers
+import io.getstream.chat.android.client.extensions.getUsersExcludingCurrent
 import io.getstream.chat.android.client.models.Channel
 import io.getstream.chat.android.core.internal.InternalStreamChatApi
 import io.getstream.chat.android.offline.ChatDomain
 
 @InternalStreamChatApi
-public fun Channel.isDirectMessaging(currentUserId: String = currentUserId()): Boolean = getUsers(currentUserId).size == 1
+public fun Channel.isDirectMessaging(currentUserId: String = currentUserId()): Boolean = getUsersExcludingCurrent(currentUserId).size == 1
```

Comment:
Looks like this is the only place where we pass in a `currentUserId` to the method being renamed. And then the `isDirectMessaging` function is almost never called with a parameter either (just once as far as I see, and I don't know if even that is necessary. Let's see if we can eliminate these params if they're not actually needed somewhere.

Reply:
We are using it inside `MessageInputView` to be sure it's initialized correctly if the view was created before a user is set. Let me check if can somehow improve that

Reply:
Maybe we can check for that inside `getUsersExcludingCurrent`, if we don't have a user, just don't filter out anything?

Reply:
👍 makes sense. Then reinitialize the view if the user was set